docs/user-guide/cli-using-sharing-team-config-files.md

```diff
@@ -0,0 +1,98 @@
+# Sharing team configuration files
+
+Team leaders can share team configuration files using several methods:
+- Shared network drive
+- Project repository (for example, GitHub)
+- Web server
+
+The following topics describe how to share the team configuration files.
+
+## Network drive
+
+1. Store the configuration files on a shared network drive.
+
+2. Issue the following command:
+
+    ```
+    zowe config import <DriveLetter>:\<SharedZoweConfig>\zowe.config.json
```

Comment:
"SharedZoweConfig" was an example of a path. For the general name maybe we want to say `<FolderPath>`?